src/wallet/coinselection.h

```diff
@@ -133,6 +133,18 @@ class COutput
     {
         return CInputCoin(outpoint, txout, input_bytes);
     }
+
+    bool operator<(const COutput& rhs) const {
+        return outpoint < rhs.outpoint;
+    }
+
+    bool operator!=(const COutput& rhs) const {
+        return outpoint != rhs.outpoint;
+    }
+
+    bool operator==(const COutput& rhs) const {
+        return outpoint == rhs.outpoint;
+    }
```

Comment:
In commit "coinselection: Use COutput instead of CInputCoin" (f2cf89eefefec98aa6f5f7a8d3f6f0abc0e0d526)

Minor: operator< above makes sense I think, but these operator== operator!= methods are only used by tests and don't really describe full equality. I'd suggest dropping these == != methods and just passing a lambda to std::mismatch in the test so it can explicitly specify the equality it checks for.

Reply:
Done in an additional commit.